src/components/SafeListSidebar/index.tsx

```diff
@@ -54,6 +48,9 @@ const SafeListSidebar = ({ children, currentSafe, defaultSafe, safes, setDefault
   }
 
   const toggleSidebar = () => {
+    if (!isOpen) {
+      trackPageEvent({ action: SAFE_NAVIGATION_EVENT, category: 'SafeListSidebar' })
```

Comment:
I think it would be better if we write the category normally instead of following PascalCase

Reply:
I added it like so because it should be easier to filter in GA dashboard.

Reply:
The last time I used analytics there were no difficulties filtering events written nicely with spaces.